src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -136,8 +143,8 @@ export default class DisabilityRatingCalculator extends React.Component {
               indx={idx}
               ratingRef={this.ratingRef}
               handleRemoveRating={this.handleRemoveRating}
-              canDelete={idx > 1}
               ref={this.setRef}
+              canDelete={this.state.ratings.length}
```

Comment:
This change is a little confusing.  The name `canDelete` implies that the prop is expecting a boolean, but you are passing a number.

Consider renaming `canDelete` to `disabled` and set it as `disabled={ratings.length < 3}`.